ledger/sandbox/BUILD.bazel

```diff
@@ -349,7 +349,6 @@ server_conformance_test(
     servers = SERVERS,
     test_tool_args = [
         "--verbose",
-        "--all-tests",
```

Comment:
If you remove `--all-tests` you can also remove some exclusions from below, I believe.

Reply:
Good catch, thank you. Didn't mean to remove it, but was a result of a rebase snafu.